ui_framework/components/modal/confirm_modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { KuiModal } from '../modal';
+import { KuiModalFooter } from './modal_footer';
+import { KuiModalHeader } from './modal_header';
+import { KuiModalBody } from './modal_body';
+import { KuiModalBodyText } from './modal_body_text';
+import { KuiButton } from '../index';
+
+export function KuiConfirmModal({ message, title, onCancel, onConfirm, cancelButtonText, confirmButtonText, ...rest }) {
```

Comment:
Would you mind putting each of these these variables on its own line? I find it much easier to scan when there are so many properties.

```javascript
export function KuiConfirmModal({
  message,
  title,
  onCancel,
  onConfirm,
  cancelButtonText,
  confirmButtonText,
  ...rest,
}) {
```